pkg/detectors/jdbc/jdbc.go

```diff
@@ -27,12 +28,32 @@ func (s Scanner) Keywords() []string {
 	return []string{"jdbc"}
 }
 
+// FromDataWithConfig will find and optionally verify Jdbc secrets in a given set of bytes, but can be configured.
+func (s Scanner) FromDataWithConfig(ctx context.Context, verify bool, data []byte, config *detectorspb.Config) (results []detectors.Result, err error) {
```

Comment:
Thinking about the `config *detectorspb.Config` part of this function. I'm wondering if it might make sense in the short term to instead do the [AWS skipIDs thing](https://github.com/trufflesecurity/trufflehog/blob/main/pkg/detectors/aws/aws.go#L20-L45), and not have to create the protobuf changes. That would also allow the precompilation of the regex I mention in my other comment.

Reply:
That does make sense. The protobuf changes made more sense when the full config plan was being implemented. Makes sense to just make the whole thing simpler for now.